compute_tools/src/compute.rs

```diff
@@ -1367,6 +1368,43 @@ LIMIT 100",
         download_size
     }
 
+    pub fn install_extension(
+        &self,
+        ext_name: &str,
+        db_name: &str,
+        ext_version: &str,
+    ) -> Result<String> {
+        let mut conf = Config::from_str(self.connstr.as_str()).unwrap();
+        conf.dbname(db_name);
+
+        let mut db_client = conf
+            .connect(NoTls)
+            .context("Failed to connect to the database")?;
+
+        let version_query = "SELECT extversion FROM pg_extension WHERE extname = $1";
+        let version: Result<String> = db_client
+            .query_one(version_query, &[&ext_name])
+            .context(format!("Failed to execute query: {}", version_query))
+            .map(|row| row.get(0));
+
+        if let Ok(installed_version) = version {
+            if installed_version == ext_version {
+                return Ok(installed_version);
+            }
+            let query = "ALTER EXTENSION $1 UPDATE TO $2";
+            db_client
+                .execute(query, &[&ext_name, &ext_version])
+                .context(format!("Failed to execute query: {}", query))?;
+        } else {
+            let query = "CREATE EXTENSION IF NOT EXISTS $1 WITH VERSION $2";
```

Comment:
`install_extension failed: Failed to execute query: CREATE EXTENSION IF NOT EXISTS $1 WITH VERSION $2`

Seems there's the same problem as in the other PR - this doesn't work with parameters.

Reply:
A second problem I found is that the version string needs to be quoted